eel/eel-editable-label.c

```diff
@@ -1339,9 +1345,18 @@ get_layout_location (EelEditableLabel  *label,
 #endif
     GtkAllocation allocation;
 
+#if !GTK_CHECK_VERSION (3, 0, 0)
     misc = GTK_MISC (label);
+#endif
     widget = GTK_WIDGET (label);
+#if GTK_CHECK_VERSION (3, 0, 0)
```

Comment:
I will test to remove the first block complete as it does nothing if the variables aren't assigned there.